python/triton/compiler.py

```diff
@@ -1092,13 +1099,26 @@ def optimize_triton_ir(mod):
     return mod
 
 
-def ast_to_ttir(fn, signature, specialization, constants, debug=False):
+def ttir_compute_capability_rewrite(mod, compute_capability):
+    if compute_capability // 10 < 9:
+        # For hardware without TMA support, we must rewrite all tile-based loads into normal loads
+        pm = _triton.ir.pass_manager(mod.context)
+        pm.enable_debug()
+        pm.add_rewrite_tiled_load_store_pass()
```

Comment:
Can we put the condition `compute_capability // 10 < 9` inside the `add_rewrite_tiled_load_store_pass` pass?

Reply:
Fixed